larray_editor/editor.py

```diff
@@ -665,6 +674,33 @@ def open_tutorial(self):
     def open_api_documentation(self):
         QDesktopServices.openUrl(QUrl("http://larray.readthedocs.io/en/stable/api.html"))
 
+    def about(self):
+        """About Editor"""
+        versions = get_versions()
+        message = \
+            """<b>LArray Editor {editor}</b>
+            <br>The Graphical User Interface for LArray. This project aims to provide an intuitive tool to 
+            visualize and manipulate labelled N-dimensional arrays. Licensed under the terms of the 
+            GNU GENERAL PUBLIC LICENSE Version 3.
+            <p>Created by Gaëtan de Menten.
+            <br>Developed and maintained by the Federal Planning Bureau (Belgium). 
+            Many thanks to all LArray regular users.
+            <p>For bug reports and feature requests, please go to our <a href="{website}">Github website</a>. 
+            For discussions around the project, please go to our <a href="{users_group}">Google Users Group</a>.
+            To be informed of each new release, please subscribe to this <a href="{announce_group}">mailing list</a>.
+            <p>Python {python} {bitness:d}bits on {system}.
```

Comment:
Add some space and a title before this more technical section. Eg. "<b>Versions of underlying libraries</b> (please copy-paste this when reporting a bug)".

Reply:
hmm, unsure we should add the "please copy-paste" comment...

Reply:
Or we could mimic Spyder? --> clicking on `Report issue...` redirect to a prefilled new issue ready to be submited.